### PR TITLE
LLM-related examples of harm

### PR DESCRIPTION
**Pull request by IshaShah27**
related to [#167](https://github.com/drivendataorg/deon/issues/167)

- Adds LLM-related examples
- Removes extra lines with `--Related academic study` when the study is prominently linked in the article (to improve readability)
- Moves one example (health algorithm that underpredicts need of black patients) from dataset bias to metric selection, which applies better.
- Changes the link to instructions for adding an example to the relevant section in `CONTRIBUTING.md` (previously it went [here](https://github.com/drivendataorg/deon/wiki/Add-a-new-item-to-the-examples-table) but didn't give instructions)

Steps used to update:
- Updated examples `deon/assets/examples_of_ethical_issues.yml`
- Ran `make docs` to update `docs/docs/examples.md`
- Updated contribution link in `docs/md_templates/_common_body_pt-BR.tpl` and `docs/md_templates/_common_body.tpl`
- Ran `make render_markdown` to update contribution link in `README.md` and `docs/docs/index.md`

Questions:
- Do we have a way to update `deon/assets/examples_of_ethical_issues_pt-BR.yml`?

### CODE REVIEW

File: deon/assets/examples_of_ethical_issues.yml [hunk 1]
```diff
@@ -126,6 +130,8 @@
       url: https://www.theverge.com/2018/3/21/17144260/healthcare-medicaid-algorithm-arkansas-cerebral-palsy
 - line_id: E.3
   links:
+    - text: Google stops Gemini, its generative AI chatbot, from generating any images from people after it began generating historically inaccurate images in an attempt to combat amplification of racial stereotypes.
```
Comment (pjbull) [inline deon/assets/examples_of_ethical_issues.yml:133]:
```suggestion
    - text: Google stops Gemini, its generative AI chatbot, from generating any images of people after it was found to generate historically inaccurate images.
```

**Review by pjbull — COMMENTED** [review]
Small wording tweak for clarity, but looks good to me!

**Comment by klwetstone** [comment]
@pjbull I made a few more changes to this -- want to take a quick look before I merge?

**Review by pjbull — APPROVED** [review]
These look good to me!

**Comment by pjbull** [comment]
I think this is good. Pretty sure just changing the yaml is the only required thing. Generated files get overwritten when PR is merged to `main`:
https://github.com/drivendataorg/deon/blob/main/.github/workflows/examples.yml#L20-L29

**Comment by ejm714** [comment]
> Is there anything else that needs to happen before I merge? Are the steps I took (in the PR description) the right ones to make sure all relevant files are updated?

For future reference, contributing doc is here: https://github.com/drivendataorg/deon/blob/main/CONTRIBUTING.md

**Comment by klwetstone** [comment]
^ Contributing doc covers adding examples and adding checklist items, but doesn't explicitly cover changing the main README so wanted to make sure

**Pull request merged**